static-site/app/styles/globals.css

```diff
@@ -0,0 +1,141 @@
+/* for debugging: add a border to all other elements on the page */
+/* body * { border: 1px solid red; } */
+
+:root {
+  --titleColor0: #4377cd; /* N */
+  --titleColor1: #5097ba; /* E */
+  --titleColor2: #63ac9a; /* X */
+  --titleColor3: #7cb879; /* T */
+  --titleColor4: #9abe5c; /* S */
+  --titleColor5: #b9bc4a; /* T */
+  --titleColor6: #d4b13f; /* R */
+  --titleColor7: #e49938; /* A */
+  --titleColor8: #e67030; /* I */
+  --titleColor9: #de3c26; /* N */
```

Comment:
Why not put these in `static-site/components/nav/styles.css`?

Reply:
My long term plan is to remove the duplicated version of this list that's currently in `static-site/src/components/splash/title.tsx` and have that component get these variables from the root.